src/RSCoin/Bank/Server.hs

```diff
@@ -301,3 +309,22 @@ serveGetExplorers sk st =
     do curPeriod <- query st GetPeriodId
        map fst . filter ((== curPeriod) . snd) <$>
            query st GetExplorersAndPeriods
+
+serveAddMintetteUsingPermit
+     :: C.WorkMode m
+     => State -> C.SecretKey -> PublicKey -> C.WithSignature (String, Int) -> ServerTESigned m ()
```

Comment:
`ServerTESigned` means that result is signed by *Bank*. It doesn't make sense to sign `()`. You'd better use `ServerTE` and to `toServer` (w/o `Signed`). I am not sure if this version works at all, because in `Communication.hs` you expect `()`, not `WithSignature ()`.